elooplib/eventloop.hpp

```diff
@@ -0,0 +1,43 @@
+#ifndef ELOOP_INCLUDED
+#define ELOOP_INCLUDED
+#include <csignal>
+#include <variant>
+#include <vector>
+#include <queue>
+#include <list>
+#include "node.hpp"
+#include "server.hpp"
+#include "client.hpp"
+
+using Host = std::variant<Server, Client>;
+
+static volatile std::sig_atomic_t stop;
+struct signalHandler
+{
+    static void handleSignal(int signal)
```

Comment:
it's ok to make it free function. But better to make it static function of EventLoop